dodo/dodo.spec

```diff
@@ -0,0 +1,31 @@
+Name:   dodo
+Version:    1.1
+Release:	1%{?dist}
+Summary:	RPM Package for dodo test runner
+
+License:    test_runner
+Source0:    %{name}-%{version}.tgz
+BuildRoot:  %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
+
+%description
+Test Runner RPM Package
```

Comment:
Functional test runner for Python based application.

Reply:
Okay.